### ISSUE TITLE
Documentation format

### ISSUE THREAD

**Issue by grimec** [ddvk/remarkable-uuuflash#4]
Wondering if the hierarchy of the docs is correct and the tool is only supported on Mac.
I'm running it on a Linux virtual machine but some processes don't seem to finish:
- recover.uuu: the process finish with "Done" but sets the device on "starting" for ever. I need to unplug it to be responsive.
- upgrade and reflash just don't return feedback and stay on 100% for ever.

![image](https://user-images.githubusercontent.com/11899229/82840163-88441600-9ec9-11ea-882a-7ab5c2db1b17.png)

**Comment by ddvk** [comment]
they work on linux the best(but havent tested a vm). 

the lack of feedback is a known issue


after recovery says done, you should be able to connect over tty

upgrade seems to works (someone else used it to fix their rootpartition), but also no feedback

**Comment by grimec** [comment]
Thank you for the quick response.
I'm trying to follow the docs but I can't make a sense out of them.
From your reply above I understand I have to run "recovery" and after that minicom, am I right?
If that's correct, I'm struggling to find the port, if I run
`dmesg | grep tty` I get back `[         0.250963] printk: console [tty0] enabled`
Am I suppose to run minicom at that port?
Also, I don't know what to do after it.
If it is "Mount flash memory" what I need to follow the hierarchy confused me as it's inside "macOs" section.

Than you for the support!

**Comment by ddvk** [comment]
yes, after recovery check the /dev directory for ttyACM0  or similar and use minicom or screen

i ll try to fix the docs

**Comment by kubycsolutions** [comment]
It would help if the readme said which platforms you  suggest running this on. Guessed, and found confirmation here, that Linux should work, but it'd help to put that up front.

**Comment by Eeems** [comment]
> It would help if the readme said which platforms you suggest running this on. Guessed, and found confirmation here, that Linux should work, but it'd help to put that up front.

There are uuu executables for linux, windows, and mac in the repository. The readme has documentation for each.

**Comment by kubycsolutions** [comment]
... it does. Comment withdrawn. I'd still consider putting that in the first or second paragraph, but I'm nitpicking.

The most basic reset didn't work for me, but the reinstall/revert did, while preserving existing data. Posted a quick recommendation on the eink discussion board. Thanks!

--
   /_  Joe Kesselman (he/him/his)
-/ _) My Alexa skill for New Music/New Sounds fans:
   /   https://www.amazon.com/dp/B09WJ3H657/

() Plaintext Ribbon Campaign
/\ Stamp out HTML mail!
________________________________
